config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
-  root :to =>'home#index'
   devise_for :users
+  root :to =>'home#index'
```

Comment:
please use updated has notation format like
`root to: "hone#index"`

Reply:
ok